src/java.base/share/classes/jdk/internal/org/objectweb/asm/ClassReader.java

```diff
@@ -296,7 +296,7 @@ public ClassReader(
             currentCpInfoOffset += cpInfoSize;
         }
         maxStringLength = currentMaxStringLength;
-        // The Classfile's access_flags field is just after the last constant pool entry.
+        // The ClassFile's access_flags field is just after the last constant pool entry.
```

Comment:
This has nothing to do with the `ClassFile` API
```suggestion
        // The Classfile's access_flags field is just after the last constant pool entry.
```

Reply:
Thanks for catching it, the refactoring was a bit more aggressive than expected.